tarea-clase-10.js

```diff
@@ -0,0 +1,211 @@
+//Simon Dice;
+let secuenciaJugador = [];
+let secuenciaMaquina = [];
+let nivel = 0;
+
+avisoInicioJugar();
+bloquerInputUsuario();
+
+document.querySelector('#iniciar').onclick = iniciarJuegoEligiendoDificultad;
+function iniciarJuegoEligiendoDificultad() {
+	const $facil = document.querySelector('#facil');
+	const $dificil = document.querySelector('#dificil');
+	if ($facil.checked) {
+		iniciarJuego();
+	} else if ($dificil.checked) {
+		iniciarJuego();
+		agregarCuadradoPurpuraYNegro();
+	} else {
+		avisoElegirDificultad();
+	}
+}
```

Comment:
a esta función ponele "validarInicioJuego"

```suggestion
function validarInicioJuego() {
	const $facil = document.querySelector('#facil');
	const $dificil = document.querySelector('#dificil');
	if (!$facil.checked && !$dificil.checked) {
		mostrarMensaje("Debe elegir una dificultad", true);
	}
	
	iniciarJuego();
	
        if ($dificil.checked) {
		agregarCuadradoPurpuraYNegro();
	}
}
```